app/config/locale/translations/ro.json

```
@@ -3,30 +3,30 @@
     "settings.locale": "ro",
     "settings.direction": "ltr",
     "emails.sender": "%s Echipa",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Verificare cont",
+    "emails.verification.hello": "Bună ziua, {{name}}",
+    "emails.verification.body": "Click pe acest link pentru a valida adresa de email.",
+    "emails.verification.footer": "Dacă nu ai cerut validarea adresei de email, poți ignora acest mesaj.",
+    "emails.verification.thanks": "Mulțumim",
+    "emails.verification.signature": "Echipa {{project}}",
+    "emails.magicSession.subject": "Login",
+    "emails.magicSession.hello": "Bună ziua",
```

```suggestion
    "emails.magicSession.hello": "Bună ziua,",
```

Please refer to the punctuation in `es.json` and make sure all the strings use the right punctuation